index.html

```diff
@@ -4246,37 +4246,39 @@ <h3>Securing Mechanism Specifications</h3>
 
         <p>
 Securing mechanism specifications that create new types of [=embedded proofs=]
-MUST specify a [=property=] for securing both [=verifiable credentials=] and
-[=verifiable presentations=]. The requirements for the property used by the
-embedded securing mechanism are as follows:
+MUST specify a [=property=] that relates the [=verifiable credential=] or [=verifiable
+presentation=] to a [=proof graph=]. 
+The requirement on the securing mechanism are as follows.
         </p>
-
         <ul>
           <li>
-The property MUST relate the [=verifiable credential=] or [=verifiable
-presentation=] to a separable and securable [=proof graph=].
-          </li>
-          <li>
-The property MUST define all terms used by the [=proof graph=]. For example,
+The securing mechanism MUST define all terms used by the [=proof graph=]. For example,
 the mechanism could define vocabulary specifications and `@context` files
 in the same manner as they are utilized by this specification.
           </li>
           <li>
-In a [=verifiable credential=], the property MUST secure the
-[=default graph=].
-          </li>
-          <li>
-In a [=verifiable presentation=], the property MUST secure the
-[=default graph=] of the [=presentation=] as well as every [=proof graph=]
-related to each [=verifiable credential=] in the [=presentation=].
+The securing mechanism MUST secure all graphs in the [=verifiable credential=] or the [=verifiable
+presentation=], except for any [=proof graphs=] securing the [=verifiable credential=] 
+or the [=verifiable presentation=] itself.
           </li>
-          <li>
+
+        </ul>
+
+      <p class="note">
+The last requirement means that the securing mechanism must secure the [=default graph=] and, 
+for [=verifiable presentations=], each [=verifiable credential=] of the presentation, together with 
+their respective [=proof graphs=].
+Also, if the verifiable presentation uses several proofs, the respective proof graphs of these proofs
+are <em>not</em> mutually secured. 
```

Comment:
I don't think this is true if chained proofs are used, where a 2nd proof has a dependency on the first proof in the verifiable presentation. 

```suggestion
```

Reply:
Well... we may have a problem that goes beyond this PR. 

1. looking at [Create proof section](https://www.w3.org/TR/vc-di-eddsa/#create-proof-eddsa-rdfc-2022) in the EdDSA spec, point (4) says that you take the hash of _transformedData_ and _proofConfig_. 
2. _proofConfig_  refers to the properties of the current proof in the making. We can put that aside for the moment. 
3. _transformedData_ refers back to the DI spec via step (3), through the concept of _unsecuredDocument_ .
4. _unsecuredDocument_ is defined in [§ 4 Algorithms](https://www.w3.org/TR/vc-data-integrity/#dfn-unsecured-data-document) as follows:

> An unsecured data document is a [map](https://infra.spec.whatwg.org/#ordered-map) with no proof [key](https://infra.spec.whatwg.org/#map-key).

This means two things:

1. I claim (but you may prove me wrong) that my interpretation is correct, including the case when proofs are chained: the mechanism only secures the original credential data, not the previous proof.
2. We do have a problem with the definition of unsecured data document, because that **_removes the proofs for  the embedded credential in the case of verifiable presentations_**. This is not what we want.

Maybe we do this in a wrong order. The spec I propose here essentially **_defines_** the concept of unsecured data, and maybe this should be the official definition and in the aforementioned section. And, if we think that the chained proofs _MUST_ be included in the unsecured data, then we have to include it in this definition explicitly (which would then be automatically valid through the reference chain).

B.t.w., do we have tests for the case of chained proofs? (cc @BigBlueHat)

Reply:
Looking at it a bit further, mathematically speaking my second statement on presentations may not be correct: in a VP, _on the top level_ there is no `proof` key in the original data, because the `proof` appear only "behind" the `verifiableCredential` key, so to say. But the definition of unsecured document is still easy to misunderstand; it does not say that it should exclude `proof` on the top level only...

Reply:
@iherman yes, the "unsecure document" language doesn't work for documents that are getting a second chained signature on them. We'll need to fix that.

Reply:
> [@msporny] @iherman ... We'll need to fix that.

New issue to track?

Reply:
I have

- Raised a separate issue (#1527)
- Made a commit removing that remark on multiple proofs.

I believe that cleanest would be to merge this PR now, and then work on solving #1517 which may force us to come back to this section, too. Leaving this PR open may lead to further complications...